extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -5,7 +5,7 @@
 
 import type * as azdataType from 'azdata';
 import * as vscode from 'vscode';
-import * as vscodeMssql from 'vscode-mssql';
+import * as mssql from '../../../mssql';
```

Comment:
This shouldn't be changed - we need to reference the vscode-mssql typings for the work I did.

You can add a separate reference to the mssql typings if needed - https://github.com/microsoft/azuredatastudio/blob/main/extensions/sql-database-projects/src/models/api/import.ts as an example